_opensearch/rest-api/snapshots/create-repository.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+title: Create Snapshot Repository
+parent: Snapshot APIs
+grand_parent: REST API reference
+nav_order: 1
+---
+
+## Create snapshot repository
+
+Registers a new repository to store snapshots. Repositories comprise two types:
+
+* File system (`fs`)
+
+* Amazon S3 bucket (`s3`)
+
+To learn more about repositories, see [Register repository]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#register-repository).
+
+
+### Path parameters
+
+Parameter | Data Type | Description
+:--- | :--- | :---
+repository | String | Repostory name. |
+
+
+### Query parameters
+
+None
+
+#### Sample request
+
+You can create repositories with `POST` or `PUT`. 
```

Comment:
PUT is for updating the repository. It creates a repository if it has not been created yet. If the repository has been created, it updates it.

Reply:
Thank you. I had tested PUT but it returned a "repository already exists." I tested  again it and now updates the repository. I'm not sure what happened originally

Reply:
I have made this change.